openfisca_france/model/prelevements_obligatoires/prelevements_sociaux/penalites.py

```diff
@@ -1,92 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
Yes, better not keep something that gives you the impression these penalties are implemented... but where to put the reference to it for future work ?
An open issue could be enough I guess.